test/NuGet.Core.Tests/NuGet.Commands.Test/ContentFileUtilsTests.cs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Linq;
+using Xunit;
+
+namespace NuGet.Commands.Test;
+
+public sealed class ContentFileUtilsTests
+{
+    [Theory]
+    [InlineData("contentFiles/cs/net45/config/config.xml", "config/config.xml")]
+    [InlineData("contentFiles/any/any/config/config.xml", "config/config.xml")]
+    [InlineData("", "")]
+    [InlineData("hello", "hello")]
+    [InlineData("a/b/", "a/b/")]
+    [InlineData("a/b/c/", "")]
+    [InlineData("a/b/c", "a/b/c")]
+    [InlineData("///config/config.xml", "config/config.xml")]
+    [InlineData("/a/b/config/config.xml", "config/config.xml")]
+    public void GetContentFileFolderRelativeToFramework(string input, string expected)
+    {
+        using var _ = new SuppressAsserts();
+
+        string actual = ContentFileUtils.GetContentFileFolderRelativeToFramework(input.AsSpan());
+
+        Assert.Equal(expected, actual);
+    }
+
+    private sealed class SuppressAsserts : IDisposable
```

Comment:
I'd like to understand why this is necessary. Is there a lot of trace noise from our tests? I'm guessing this is suppressing the 2 ` Debug.Assert` we have in `ContentFileUtils`? 
If so, then:
1. Is it a preferable practice that we not to have the asserts and use a different approach? Are we doing something unusual?
2. If it's ok to have the asserts, then I'd like to have this suppression more globally, somehow.

Reply:
Without this the test blocks on a dialog, courtesy of the default trace listeners.

Either option you outline seems good to me. I don't have enough context on why the assert is there. It seems harmless enough, especially as it's debug only.